docs/snippets/common/my-component-story-with-storyname.js.mdx

```diff
@@ -1,9 +1,16 @@
 ```js
 // MyComponent.story.js | MyComponent.story.jsx | MyComponent.story.ts | MyComponent.story.tsx
 
-export const Simple = () => <MyComponent />;
+import MyComponent from './MyComponent';
+
+export default {
+  component: MyComponent,
+};
+
+export const Simple = {
+  decorators: [...],
+  parameters: {...},
+};
 
 Simple.storyName = 'So simple!';
```

Comment:
Should this be able to be included in the above `Simple` object?

Reply:
This one is story-based. It relates to [this](https://storybook.js.org/docs/react/api/csf) portion of the documentation. And in there the decorators and parameters are being used within a story. It indirectly reinforces the idea that decorators/parameters can be used within the stories themselves.

Reply:
Oh yes, I agree! I was wondering if `Simple.storyName` could also be included in the object?

Reply:
To the best of my knowledge, it doesn't. I experimented with it and sadly it did not rename the story 😞!

Reply:
Sad. Maybe @shilman knows why? I would assume it would work as everything else, but if it doesn't, at least we have it documented 😄

Reply:
Sorry just getting to this now. In CSF3 it's:

```js
export const Simple = {
  name: 'So simple!'
  ...
}
```

This matches CSF1:

```js
export const Foo = () => ...
Foo.story = {
  name: 'bar'
}
```

However, when we hoisted this in CSF2 we couldn't use `name` because it already has meaning in JS, so we had to call it `storyName`.

It's confusing to go back and forth like this, but hopefully people adopt CSF3 and we can put `storyName` behind us. WDYT?

Reply:
I agree -- if we move everything to CSF3, let's use the `name` property and put the final nail in the coffin of `storyName` 👍

Reply:
Same here! I'm on board with the `name` property and consolidate all properties within the story construct. It makes it leaner.